snapcraft/plugins/python2.py

```diff
@@ -144,7 +153,7 @@ def dist_packages_dir(self):
 
     @property
     def python_version(self):
-        return self.run_output(['pyversions', '-i'])
+        return self.run_output(['pyversions', '-d'])
```

Comment:
Yeah, I started seeing this strange issue with py3versions and ended up finding out about this by reading the source code.